train.py

```diff
@@ -0,0 +1,96 @@
+"""Script for training a tensorflow model on audio datasets.
+
+Usage:
+    python train.py --dataset=fma
+"""
+
+import argparse
+import logging
+
+import tensorflow as tf
+
+from datasets import fma
+
+MINI_BATCH_SIZE = 32
+LEARNING_RATE = .5
+
+
+def load_data(dataset, split="training", size="small"):
+    """Load data into trainable iterator."""
+    logging.info("Loading dataset %s" % dataset)
+    # TODO(amy): build support for cross validation rather than
+    # static train/test split
+    if dataset == 'fma':
+        path_to_data = 'raw_data'
+        dataset, all_labels = fma.read(path_to_data, split, size)
+        # Shuffle with shuffle buffer 10x the batch size
+        dataset = dataset.shuffle(MINI_BATCH_SIZE * 10)
+        batched_dataset = dataset.batch(MINI_BATCH_SIZE)
+        # Create iterator that will make one pass through the dataset
+        iterator = batched_dataset.make_one_shot_iterator()
+        # Returns placeholder tensors (features and labels) representing
+        # one minibatch in the single pass
+        x, y = iterator.get_next()
+        # Extract mfcc features and genre as labels
+        features = x['mfcc']
+        labels = y['genre_top']
+    else:
+        raise ValueError('Unknown dataset %s' % dataset)
+    return features, labels, all_labels
+
+
+# TODO(amy): figure out how we want to handle flexibly specifying the
+# network structure
+def predict(features, num_labels):
+    """Construct actual network to predict labels from features"""
+    # create a single layer network returning unnormalized probability
+    # of each class
+    logits = tf.contrib.layers.fully_connected(
+        features, num_labels, activation_fn=tf.nn.relu)
+    return logits
+
+
+def specify_model(features, labels, all_labels, num_oov_buckets=1):
+    """Specify the loss and structure of the model."""
```

Comment:
Same comment RE: docstring. I think it helps readability a lot to have a full-blown one, sicne it's easy to forget what things are -- e.g. that `labels` is a string tensor or already int, etc. 

```
Args:
  features: A [batch_size, input_feature_dimension] tf.float32 tensor of features to predict from.
  labels: A [batch_size] tf.string tensor of labels.
  all_labels: A Python list of strings containing all the labels in the dataset.
  num_oov_buckets: The number of out-of-vocabulary buckets to create for handling labels that are not in all_labels.

Returns:
  train_step: A tf.Operation that minimizes the loss.
  loss: A scalar tf.float32 tensor that is the total loss for the current mini-batch.
```